Source/BugsnagCrashReport.m

```diff
@@ -671,12 +671,9 @@ + (instancetype)errorDataFromThreads:(NSArray *)threads {
             }
             NSString *contentValue = data[@"value"];
 
-#pragma clang diagnostic push
-#pragma ide diagnostic ignored "OCDFAInspection"
```

Comment:
I couldn't find anything online on what this actually did, but the compiler warning implied that it was ignored